hydrus/data/crud.py

```diff
@@ -56,6 +56,10 @@ def insert(object_, id_=None):
     """Insert an object to database [POST] and returns the inserted object."""
     # NOTE: We are inserting the object, no need to check if similar one already exists.
     #       Data can be redundant/identical, they must have different "@id"
+
+    # TODO: @chrizandr You're not setting instance.type_ and you're not using keymap
+    # to dereference class types. Try any other category other then thermal for insertion
+    # Also, make sure crud.py passes all the tests in test_db.py (Currently all failing)

```

Comment:
You asked for `instance.type_`, the `Graph.type_` attribute is not for the anyone to modify. The ORM handles it in the back. What is your doubt?